rust/macros/lib.rs

```diff
@@ -189,3 +190,9 @@ pub fn vtable(attr: TokenStream, ts: TokenStream) -> TokenStream {
 pub fn concat_idents(ts: TokenStream) -> TokenStream {
     concat_idents::concat_idents(ts)
 }
+
+/// `pin_data!` convenience attribute
```

Comment:
```suggestion
/// `pin_data!` convenience attribute.
///
/// It allows to avoid having to wrap struct definitions in `pin_data! { ... }`.
///
/// # Examples
///
/// ```rust
/// #[pin_data]
/// struct A;
/// ```
```

Reply:
Given `pin_data!` will be hidden, I would avoid mentioning it at all.